### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by carohadad**

### CODE REVIEW

File: docs/Discover Tella/accesibility.md
Comment (djordjemarusic) [inline docs/Discover Tella/accesibility.md:1]: The name of the file is a typo too, but we don't need to correct that.

Comment (djordjemarusic) [inline docs/Discover Tella/accesibility.md:1]: There is a typo in the name of the file, but we don't need to correct that.

**Review by djordjemarusic — APPROVED** [review]

File: docs/Discover Tella/accesibility.md
Comment (carohadad) [reply docs/Discover Tella/accesibility.md:1]: yes! but i cannot change that until we load translations because otherwise it will all get messed up!

Comment (carohadad) [reply docs/Discover Tella/accesibility.md:1]: We cannot for now :(

**Pull request merged**